src/test/scenarios/mixed-reality/output/src/mixed-reality/azext_mixed_reality/manual/__init__.py

```diff
@@ -0,0 +1,12 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+__path__ = __import__('pkgutil').extend_path(__path__, __name__)
```

Comment:
Do we need to add that manual folder __init__.py in that incremental generated case?

Reply:
> Do we need to add that manual folder **init**.py in that incremental generated case?

Any reason not to include? it will still let customer override some generated code if needed

Reply:
Then I don't see the logic of loading the manual folder for commands or params or help

Reply:
> Then I don't see the logic of loading the manual folder for commands or params or help

fixed